lib/ansible/modules/cloud/amazon/cloudfront_invalidation.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/python
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: cloudfront_invalidation
+short_description: create invalidations for aws cloudfront distributions
+description:
+    - Allows for invalidation of a batch of paths for a CloudFront distribution.
+requirements:
+  - boto3 >= 1.0.0
+  - python >= 2.6
+version_added: "2.4"
+author: Willem van Ketwich (@wilvk)
+options:
+    distribution_id:
+      description:
+        - The alias of the cloudfront distribution.
+          Can be specified instead of distribution_id.
+      required: false
+    caller_reference:
+      description:
+        - The id of the cloudfront distribution.
+      required: false
+      default: current datetime stamp
+    invalidation_batch:
+      description:
+        - A list of paths on the distribution to invalidate.
```

Comment:
Also, in these docs can you add a note that wildcards are allowed, like `/foo/bar/*`?